Mathlib/Geometry/Manifold/IsManifold/Basic.lean

```diff
@@ -892,8 +892,7 @@ instance empty [IsEmpty M] : IsManifold I n M := by
     _ = (e.symm.source) ∩ ∅ := by rw [preimage_empty]
     _ = ∅ := inter_empty e.symm.source
   have : t = ∅ := calc t
-    _ = I.symm ⁻¹' (e.symm ≫ₕ e').source ∩ range I := by
-      rw [← Subtype.preimage_val_eq_preimage_val_iff]
+    _ = I.symm ⁻¹' (e.symm ≫ₕ e').source ∩ range I := by order
```

Comment:
To be honest, I'm not even sure how `order` manages to prove this. My initial reaction is "this is not clearer".

Perhaps you can tell me why this is wrong, but otherwise I'd rather revert this.